src/Model/EducationRoot.php

```diff
@@ -22,12 +22,42 @@
 * @license   https://opensource.org/licenses/MIT MIT License
 * @link      https://graph.microsoft.com
 */
-class EducationRoot extends Entity
+class EducationRoot implements \JsonSerializable
```

Comment:
Did notice that too don't know why the change was made.

Reply:
This is expected. EducationRoot is the type of a singleton. It was misdefined as an entityset. Since there can only be one educationRoot, no need for an identifier.

Reply:
Thanks for clarifying